### PR TITLE
WinUI-2902 : Prepare Precision content in the rating control in UG

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Prepare precision content in the rating control in UG.

### CODE REVIEW

File: winui/Rating/Precision.md [hunk 1]
```diff
@@ -10,3 +10,86 @@ documentation: ug
 # Precision in WinUI Rating
 
 This section explains the precision features available in the WinUI Rating control.
+
```
Comment (Manivannan-E) [inline winui/Rating/Precision.md:13]: @Priyanka-Vijayasankar  Learn here all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact.

File: winui/Rating/Precision.md [hunk 2]
```diff
@@ -10,3 +10,86 @@ documentation: ug
...
+## Rating Control using the Precision property
+
+The Rating control Precision can be customized using the `Precision` Property. It have three types Full, Half and Exact. The default value is "Full".
```
Comment (Manivannan-E) [inline winui/Rating/Precision.md:16]: @Priyanka-Vijayasankar The rating value can be provided with three options such as full, half, and exact using the `Precision` property. The default value is `Full`.

File: winui/Rating/Precision.md [hunk 3]
```diff
@@ -10,3 +10,86 @@ documentation: ug
 # Precision in WinUI Rating
 
 This section explains the precision features available in the WinUI Rating control.
+
+## Rating Control using the Precision property
+
+The Rating control Precision can be customized using the `Precision` Property. It have three types Full, Half and Exact. The default value is "Full".  
+
+## Full
+
```
Comment (Manivannan-E) [inline winui/Rating/Precision.md:19]: @Priyanka-Vijayasankar Rating value can be provided with whole precision.

File: winui/Rating/Precision.md [hunk 4]
```diff
@@ -10,3 +10,86 @@ documentation: ug
...
+//Rating value setted with Exact Precision
+rating.Value = 3.7;
+rating.Precision = RatingPrecision.Exact;
+rating.ItemsCount = 5;
+
+{% endhighlight %}
+{% endtabs %}
+
+[Rating control](Rating_images/winui_rating_precisionexact.png)
```
Comment (Manivannan-E) [inline winui/Rating/Precision.md:94]: @Priyanka-Vijayasankar Change image name like winui_rating_precision_exact

File: winui/Rating/Precision.md [hunk 5]
```diff
@@ -10,3 +10,86 @@ documentation: ug
...
+     Precision="Exact"
+     ItemsCount="5">
+</syncfusion:SfRating>
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfRating rating = new SfRating();
+
+//Rating value setted with Exact Precision
```
Comment (Manivannan-E) [inline winui/Rating/Precision.md:86]: @Priyanka-Vijayasankar Remove the comments in code snippet.

File: winui/Rating/Precision.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:13]: Changed the above description as mentioned above.

File: winui/Rating/Precision.md (hunk 3 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:19]: Added the above mentioned content.

File: winui/Rating/Precision.md (hunk 4 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:94]: Changed the image name as mentioned above.

File: winui/Rating/Precision.md (hunk 5 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:86]: Removed the comments in the code snippet.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Rating/Overview.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ documentation: ug
 
 The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service provided, such as movies applications, products, and more.
```
Comment (OnyimbiNelson) [inline winui/Rating/Overview.md:12]: The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service, such as movie applications, products, and more.

File: winui/Rating/Precision.md [hunk 7]
```diff
@@ -1,12 +1,88 @@
 ---
 layout: post
 title: Precision with WinUI Rating control | Syncfusion
-description: Learn here all about precision mode defines the accuracy level and it has full, half and exact options of the SfRating control.
+description: Learn here all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact.
```
Comment (OnyimbiNelson) [inline winui/Rating/Precision.md:4]: description: Learn all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact here.

File: winui/Rating/Overview.md (hunk 6 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Overview.md:12]: changed the content as mentioned above.

File: winui/Rating/Precision.md (hunk 7 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:4]: Changed the content as mentioned above.

**Pull request merged**